app/src/main/java/org/oppia/android/app/devoptions/ForceCrashListener.kt

```diff
@@ -0,0 +1,6 @@
+package org.oppia.android.app.devoptions
+
+/** Listener for force crashing the app. */
```

Comment:
`Listener for Force Crash TextView click. `

Reply:
Small adjustment to Rajat's:

``Listener for when the 'force crash' button is clicked.``

(note capitalization)

Reply:
Done.